### DISCUSSION TITLE
The @click are not working

### DISCUSSION THREAD

**Discussion by tinaaa071** [slidevjs/slidev#1457] [Q&A] [answered]
Hello, after I installed Slidev and opened the project, I found all of my `@click` functions were not working.
Can anyone help me to check what happened to my project?
Is there anything I'm missing?
I'm not very good at coding😢
Many thanks!

[My project](https://github.com/tinaaa071/Slidev)
-
Update:
After investigating for 2 hrs, I found all of my slides were covered by a `<div>`, and its position is `absolute`.
I can't find where the `<div>` is coming from, and how to remove it.
![1711053601452](https://github.com/slidevjs/slidev/assets/122771951/9c9b1ea1-5200-4aa3-8b34-116b6affe8cc)

**Comment by kermanx** [comment, marked as answer]
Resolved via #1459
